### ISSUE TITLE
Secondary services

### ISSUE THREAD

**Issue by philips77** [NordicSemiconductor/Android-nRF-Connect#76]
As a user, I'd like to see and use Secondary services, despite no one is using them. So far they are ignored by the app.

**Comment by bunalti** [comment]
Bump! 

> As a user, I'd like to see and use Secondary services, despite no one is using them. So far they are ignored by the app.


Hello do you know is this issue still persists? I am unable to see secondary services listed in nrfConnect on my ESP-IDF BLE app; when I change my attribute table to primary service it is shown but not in secondary

**Comment by philips77** [comment]
Yes, sorry for that. Secondary services is still one of the biggest features that await implementing...
